js/admin.js

```diff
@@ -110,6 +110,7 @@ jQuery( document ).ready(function($) {
 							}
 						}
 					};
+					$('.progress-bar').addClass('install-error');
```

Comment:
I don't think this is not needed. The error is only show if the composer files are not installed.